pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -423,12 +426,12 @@ func NewNodeLifecycleController(
 			if !isPod {
 				deletedState, ok := obj.(cache.DeletedFinalStateUnknown)
 				if !ok {
-					klog.Errorf("Received unexpected object: %v", obj)
+					logger.Error(nil, "Received unexpected object", "obj", obj)
```

Comment:
Please replace "obj" with "object" in the entire PR (https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments).

Reply:
done